### DISCUSSION TITLE
403 Forbidden after installation using Softaculous

### DISCUSSION THREAD

**Discussion by roslantangah** [YOURLS/YOURLS#3000] [Q&A] [answered]
Hello,

I have problem after doing installation using softaculous from cpanel. This appear in the browser:

> **Forbidden**
> You don't have permission to access this resource.
> 
> Additionally, a 500 Internal Server Error error was encountered while trying to use an ErrorDocument to handle the request.

Thank you.

**Comment by dgw** [comment, marked as answer]
Contact whoever maintains the Softaculous installer? We aren't involved.
